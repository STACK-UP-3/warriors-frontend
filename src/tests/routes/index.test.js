import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../../routes/index'

test('test render routes correctly',()=>{
    const wrapper = shallow(<Routes/>)
    expect(wrapper).toMatchSnapshot();
})