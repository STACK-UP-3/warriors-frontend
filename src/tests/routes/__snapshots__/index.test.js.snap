// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render routes correctly 1`] = `
<BrowserRouter>
  <div
    className="app"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/dashboard"
      />
      <Route
        component={[Function]}
        path="/login"
      />
      <Route
        component={[Function]}
        path="/signup"
      />
      <Route
        component={[Function]}
        path="/reset"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(Profile)",
            "type": [Function],
          }
        }
        path="/profile"
      />
    </Switch>
  </div>
</BrowserRouter>
`;
